announcement/announcement-api/api/src/bundle/announcement_eu.properties

```diff
@@ -25,6 +25,7 @@ java.alert.youacc.pes=Baimenik ez duzu ohar hau eskuratzeko\: - {0}
 java.alert.thisis=Beste erabiltzaile bat item hau editatzen ari da. Saiatu, arren, berriro geroago.
 ### (moot) java.alert.youcreate = you don't have permissions to create this announcement -
 java.alert.youchoosegroup=Talde bat aukeratu behar duzu, gutxienez.
+java.alert.youchooserole=Rola bat aukeratu behar duzu, gutxienez.
```

Comment:
Put all the new strings together at the end of the file, put a comment for S2U-36.